tests/parametric/test_parametric_endpoints.py

```diff
@@ -0,0 +1,732 @@
+# TODO(mabdinur): Move this file to tests/test_the_test/
+"""
+This module provides one unit test for each parametric endpoint.
+The results of these unit tests are reported to the feature parity dashboard.
+Parametric endpoints that are not tested in this file are not yet supported.
+Avoid using these endpoints in the parametric tests.
```

Comment:
Maybe add a link to the docs showing how to startup the python server and view the endpoints?  https://github.com/DataDog/system-tests/blob/main/docs/scenarios/parametric.md#http

Reply:
YES!! I will add this